LICENSE

```diff
@@ -41,7 +41,7 @@ Also available at: https://www.apache.org/licenses/LICENSE-2.0
 
                               Apache License
                         Version 2.0, January 2004
-                     http://www.apache.org/licenses/
+                     https://www.apache.org/licenses/
```

Comment:
I am not sure if this change and the similar change to the GPL header constitute changes to the licenses in question: if so, that might be problematic.